src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -724,14 +801,7 @@ public bool Verify()
         /// exceptions that can be thrown.
         /// </para>
         /// <para>
-        ///   The SubjectPublicKeyInfo from the certificate determines what PEM labels are accepted for the private key.
-        ///   For RSA certificates, accepted private key PEM labels are "RSA PRIVATE KEY" and "PRIVATE KEY".
-        ///   For ECDSA certificates, accepted private key PEM labels are "EC PRIVATE KEY" and "PRIVATE KEY".
-        ///   For DSA certificates, the accepted private key PEM label is "PRIVATE KEY".
-        /// </para>
-        /// <para>PEM-encoded items that have a different label are ignored.</para>
-        /// <para>
-        ///   Combined PEM-encoded certificates and keys do not require a specific order. For the certificate, the
+        ///   The Subjbool restrictPublicKeyToEncryption PEM-encoded certificates and keys do not require a specific order. For the certificate, the
         ///   the first certificate with a CERTIFICATE label is loaded. For the private key, the first private
```

Comment:
This doesn't seem intentional...  Especially not "The subjbool restrictPublicKeyToEncryption"

Reply:
... 
![Doh](https://user-images.githubusercontent.com/361677/94321725-713c3000-ff5e-11ea-9db9-d34f9e482112.png)